src/SuperSelectField.js

```diff
@@ -406,10 +429,35 @@ class SelectField extends Component {
     event.preventDefault()
     const { selectedItems } = this.state
     if (this.props.multiple) {
-      const selectedItemExists = selectedItems.some(obj => areEqual(obj.value, selectedItem.value))
-      const updatedValues = selectedItemExists
-        ? selectedItems.filter(obj => !areEqual(obj.value, selectedItem.value))
-        : selectedItems.concat(selectedItem)
+      const { allowSelectAll, selectAllItem } = this.props;
+      let updatedValues;
+      if (allowSelectAll && selectedItem.value === selectAllItem.value) {
```

Comment:
What is `selectAllItem` ? Can you document it in Readme.md please ? especially its value property.

Is it a bool ? if not, can you give a usecase please.